pages/treeherder.py

```diff
@@ -309,13 +415,24 @@ def pin_job(self):
             self.wait.until(EC.visibility_of(el))
             el.click()
 
+        def select_next_panel_tab(self):
+            el = self.find_element(*self._job_details_panel_locator)
+            el.send_keys('t')
+
     class Pinboard(Region):
 
         _root_locator = (By.ID, 'pinboard-panel')
+        _classification_comment_locator = (By.ID, 'classification-comment')
         _clear_all_menu_locator = (By.CSS_SELECTOR, '#pinboard-controls .dropdown-menu li:nth-child(4)')
         _jobs_locator = (By.CLASS_NAME, 'pinned-job')
         _open_save_menu_locator = (By.CSS_SELECTOR, '#pinboard-controls .save-btn-dropdown')
         _pinboard_remove_job_locator = (By.CSS_SELECTOR, '#pinned-job-list .pinned-job-close-btn')
+        _pinboard_related_bugs_locator = (By.CSS_SELECTOR, '.pinboard-related-bugs-btn a em')
+
+        @property
+        def is_classification_comment_not_empty(self):
+            el = self.find_element(*self._classification_comment_locator)
+            return 'ng-not-empty' in el.get_attribute('class')
```

Comment:
This implementation detail is concerning. If we changed the name of this class, the test will probably never fail. I would be more comfortable if we had at least one test for each potential return value from this property.